packages/insomnia-inso/src/global.d.ts

```diff
@@ -0,0 +1,8 @@
+import { Database } from './db';
+
+declare module 'insomnia-send-request' {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- TODO(TSCONVERSION) missing types from insomnia-send-request
+  export function getSendRequestCallbackMemDb(environmentId: string, db: Database): Promise<any>;
```

Comment:
Do we want `Promise<UNKNOWN>` here instead of `Promise<any>`?

Reply:
I tried exactly that, but unfortunately because of a gotcha with how ambient type declarations work it has trouble if we were to use `UNKONWN` because we have to _import_ that from another module in the project which is not allowed.

I went with it for now and left the TSCONVERSION comment because this whole file is gonna go away anyway once we get better conversions so I didn't feel it was worth the time.